{
  "id": "uno-ooo-parser",
  "version": "0.2.15",
  "libre_office_ver": "7.4",
  "name": "InteractiveNetworkOffLineException",
  "type": "exception",
  "namespace": "com.sun.star.ucb",
  "parser_args": {
    "sort": false,
    "long_names": true,
    "remove_parent_inherited": true
  },
  "writer_args": {
    "include_desc": true
  },
  "data": {
    "allow_db": true,
    "from_imports": [
      [
        ".interactive_network_exception",
        "InteractiveNetworkException",
        "InteractiveNetworkException_7ca31159"
      ]
    ],
    "from_imports_typing": [],
    "extends_map": {
      "com.sun.star.ucb.InteractiveNetworkException": "InteractiveNetworkException_7ca31159"
    },
    "quote": [],
    "typings": [],
    "requires_typing": false,
    "full_imports": {
      "general": [
        "com.sun.star.ucb.InteractiveNetworkException"
      ],
      "typing": []
    },
    "imports": [],
    "name": "InteractiveNetworkOffLineException",
    "namespace": "com.sun.star.ucb",
    "extends": [
      "com.sun.star.ucb.InteractiveNetworkException"
    ],
    "desc": [
      "A network error specifying an off line condition."
    ],
    "url": "https://api.libreoffice.org/docs/idl/ref/exceptioncom_1_1sun_1_1star_1_1ucb_1_1InteractiveNetworkOffLineException.html",
    "items": {}
  }
}